pkg/mount/block/blockdev_linux.go

```diff
@@ -604,27 +654,27 @@ func (b BlockDevices) FilterBlockPCI(blocklist pci.Devices) BlockDevices {
 	return partitions
 }
 
-// parsePCIBlockList parses a string in the format vendor:device,vendor:device
-// and returns a list of PCI devices containing the vendor and device pairs to block.
-func parsePCIBlockList(blockList string) (pci.Devices, error) {
+// parsePCIList parses a string in the format vendor:device,vendor:device
+// and returns a list of PCI devices containing the vendor and device pairs.
+func parsePCIList(parseList string) (pci.Devices, error) {
 	pciList := pci.Devices{}
-	bL := strings.Split(blockList, ",")
+	bL := strings.Split(parseList, ",")
 	for _, b := range bL {
 		p := strings.Split(b, ":")
 		if len(p) != 2 {
-			return nil, fmt.Errorf("BlockList needs to be of format vendor1:device1,vendor2:device2...! got %v", blockList)
+			return nil, fmt.Errorf("%w: %q", ErrListFormat, parseList)
```

Comment:
same thing here, it's the convention, almost always, to have
%q:%w
see just about every go program out there.
or codesearch
%s.*%w f:.*.go

https://cs.opensource.google/search?q=%25s.*%25w%20f:.*.go&sq=&ss=go

The %q is just insulation against problematic file names ..

Reply:
Done.